recipe/build.sh

```diff
@@ -21,4 +21,5 @@ then
     export CUDA_INCLUDE_PATH="${PREFIX}/include,${CUDA_HOME}/include,${CONDA_PREFIX}/include"
 fi
 
+export BUILD_VERSION=${PKG_VERSION}
```

Comment:
Is this change needed?

Reply:
Yeah, without it torchvision.__version__ shows 0.7.0a0, which isn't a real tag in GitHub (it also isn't the correct version number).